ios/Source/Core/LRBlockCallback.h

```diff
@@ -13,18 +13,19 @@
  */
 
 #import "LRCallback.h"
+#import "LRCookieCallback.h"
 
 typedef void (^LRFailureBlock)(NSError *);
 typedef void (^LRSuccessBlock)(id);
 
 /**
  * @author Bruno Farache
  */
-@interface LRBlockCallback : NSObject <LRCallback>
+@interface LRBlockCallback : NSObject <LRCallback, LRCookieCallback>
```

Comment:
This is weird: `LRBlockCallback` is a generic object that converts a delegate into two blocks. "Cookie" here doesn't make sense

Reply:
The purpose of this was to avoid creating a similar class LRBlockCookieCallback to do this